src/main/kotlin/com/cjbooms/fabrikt/generators/controller/ControllerGeneratorUtils.kt

```diff
@@ -0,0 +1,85 @@
+package com.cjbooms.fabrikt.generators.controller
+
+import com.cjbooms.fabrikt.generators.model.JacksonModelGenerator.Companion.toModelType
+import com.cjbooms.fabrikt.model.BodyParameter
+import com.cjbooms.fabrikt.model.ControllerType
+import com.cjbooms.fabrikt.model.IncomingParameter
+import com.cjbooms.fabrikt.model.KotlinTypeInfo
+import com.cjbooms.fabrikt.model.RequestParameter
+import com.cjbooms.fabrikt.util.KaizenParserExtensions.safeName
+import com.reprezen.kaizen.oasparser.model3.Operation
+import com.reprezen.kaizen.oasparser.model3.Parameter
+import com.reprezen.kaizen.oasparser.model3.Response
+import com.squareup.kotlinpoet.TypeName
+import com.squareup.kotlinpoet.asTypeName
+
+object ControllerGeneratorUtils {
+    /**
+     * This maps the OpenAPI response code with a ClassName object.
+     * Pulling out the first response. This assumes first is happy path
+     * may need to revisit if we want to have conditional responses
+     */
+    fun Operation.defaultResponse(basePackage: String): Pair<Int, TypeName> {
+        // Map of response code to nullable name of schema
+        val responseDetails = defaultResponseObject()
+        return responseDetails.first to (responseDetails
+            .second
+            .contentMediaTypes
+            .map { it.value?.schema }
+            .filterNotNull()
+            .firstOrNull()
+            ?.let { toModelType(basePackage, KotlinTypeInfo.from(it)) }
+            ?: Unit::class.asTypeName())
+    }
+
+    fun Operation.defaultResponseObject(): Pair<Int, Response> {
+        val toResponseMapping = responses
+            .filter { it.key != "default" }
```

Comment:
Even more reason to rename the functions away from default, it's an overloaded term!